docs/products/clickhouse/concepts/clickhouse-tiered-storage.md

```diff
@@ -106,6 +108,31 @@ between the two layers.
     connect to an external existing
     object storage or cloud storage bucket.
 
+-   There can be a mismatch in the displayed amount of data in object storage between:
+
+    - [Aiven Console](https://console.aiven.io/) > Aiven for ClickHouse service >
+      **Tiered storage**
+
+      and
+
+    - [Aiven Console](https://console.aiven.io/) > Aiven for ClickHouse service >
+      **Databases and tables** > <ConsoleLabel name="actions"/> > **View details** >
+      **Storage details**
```

Comment:
I really advise caution about this "micro docs" inside a given doc. The day we touch up the UI to move a section, it'll be very hard to update all these little procedures. Ideally, we really want something like: 

> ... between your Clickhouse [Tiered storage]() and [Storage details]().

If ppl have no clue what these are and how to display these views, they just click on the links. 
This also helps us lighten up the content and ensure a pattern in each feature such as having a task to display tiered storage, storage details, and the rest.
For example, this reveals we should perhaps have a section to explain and display tierd storage in https://aiven.io/docs/products/clickhouse/howto/list-tiered-storage?